### PR TITLE
[WFLY-15559]:Move WildFly Preview to a native Jakarta namespace varia…

### PR DESCRIPTION
**Pull request by jimma**
…nt of the webservices subsystem module

https://issues.redhat.com/browse/WFLY-15559

### CODE REVIEW

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: ee-9/pom.xml [hunk 1]
```diff
@@ -2675,6 +2675,42 @@
...
+                <groupId>org.jboss.ws</groupId>
+                <artifactId>jbossws-spi</artifactId>
+                <version>${preview.version.org.jboss.ws.spi}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
```
[the remark this answers is not in the document]
Comment (jimma) [reply ee-9/pom.xml:2712]: These dependencies added are for preview/jakarta versions. Will moving out these dependencies still work for WildFly Preview?

[the remark this answers is not in the document]
Comment (bstansberry) [reply ee-9/pom.xml:2712]: Yes. The declarations you have here will be what WFP uses.

https://lists.jboss.org/archives/list/wildfly-dev@lists.jboss.org/thread/A2ULGAF2VGAONJ272L6N5I44QVVPSYLH/ describes how the various 'internal boms' work.  What you've done above is configure '5) An 'wildfly-preview' bom'. What I'm looking for is for the standard WildFly dependencies to be moved out of the 'common bom' (#2) and into the '3) An 'ee' bom for standard WildFly dependencies that get installed via the
wildfly-ee feature pack and that are not used in WFP'.

File: ee-9/common/pom.xml [hunk 2]
```diff
@@ -1218,6 +1218,17 @@
...
+        <dependency>
+            <groupId>${ee.maven.groupId}</groupId>
+            <artifactId>wildfly-webservices-server-integration-jakarta</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```
[the remark this answers is not in the document]
Comment (jimma) [reply ee-9/common/pom.xml:1230]: Fixed.

File: ee-9/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jimma) [reply ee-9/pom.xml:2712]: Thanks @bstansberry. I wrongly read to moving out dependencies from ee-9/pom.xml to ee-feature-pack/pom.xml.  
Already moving out these dependencies from root pom to ee-feature-pack  pom file.

**Comment by bstansberry** [comment]
Thanks @jimma

**Pull request merged**